pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -1501,11 +1501,6 @@ in
     dependencies = with self; [ nvim-treesitter plenary-nvim ];
   };
 
-  render-markdown-nvim = super.render-markdown-nvim.overrideAttrs {
-    dependencies = with self; [ nvim-treesitter ];
```

Comment:
Shouldn't this be `nvim-treesitter-parsers.markdown nvim-treesitter-parsers.latex`?

Reply:
I think that this wouldn't be enough to get the parsers "installed" properly.

Reply:
`nvim-treesitter.withPlugins` just adds those to `nvim-treesitter.passthru.dependencies`, so it should be enough

Reply:
Oh, I didn't know.